uiowa/uiowa
# [Facilities] building page theming

Resolves https://github.com/uiowa/uiowa/issues/7327

# How to test

```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local fm-buildings && ddev drush @facilities.local uli /building/0006
```
- Import building hours `ddev drush @facilities.local fm-buildings` 
- Review code
- Test out all variations of bathrooms and confirm that they appear and don't display empty headings
- Test that alerts appear below the banner in a card format with gray background
- Confirm that accessibility links work

## Unresolved and possible follow-up issue
- [ ] Layout with card sidebar renders when there is no content https://facilities.uiowa.ddev.site/building/0147
